src/utils/currencies.js

```diff
@@ -1,5 +1,5 @@
 /* Downloaded XML file from https://www.currency-iso.org/en/home/tables/table-a1.html and converted to javascript */
-export default {
+const currencies = {
```

Comment:
Was there a specific reason for this change? I'm curious why this was necessary.